libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/OfflineRouteFoundCallback.java

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.services.android.navigation.ui.v5;
+
+import android.support.annotation.NonNull;
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute;
+import com.mapbox.services.android.navigation.v5.navigation.OfflineError;
+import com.mapbox.services.android.navigation.v5.navigation.OnOfflineRouteFoundCallback;
+
+class OfflineRouteFoundCallback implements OnOfflineRouteFoundCallback {
```

Comment:
Is this necessary? Why not just have `NavigationViewRouter` implement `OnOfflineRouteFoundCallback`?

Reply:
This was done on purpose for isolated unit testing